### DISCUSSION TITLE
installing defdap

### DISCUSSION THREAD

**Discussion by ZheningYang1** [MechMicroMan/DefDAP#108] [General]
Hi, we tried to install defdap along with numpy 1.26 and matplotlib 3.8.0, numpy 1.26 does not have the numpy.float function and so we got an error when trying to load the packages into the notebook.
So we tried using matplotlib 3.3.2 and numpy 1.19.2 (known to work with defdap) but there was an error about having permissions.

has anyone got any suggestions about what we can do to solve this?

thanks in advance :)

**Comment by mikesmic** [comment]
I think this has been fixed on the master branch but it may not be released yet. I'll check and make a release when I get chance. A quick fix is to run,
```
np.int = int
np.float = float
```
after you import numpy.
